scripts/perf-test.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+NUM_RECORDS=${NUM_RECORDS:-10000000}
+RECORD_SIZE=${RECORD_SIZE:-1024}
+ON_SHUTDOWN=()
+PERF_TEST=${SCRIPT_DIR}/perf-tests
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NOCOLOR='\033[0m'
+
+. "${SCRIPT_DIR}/common.sh"
+KROXYLICIOUS_CHECKOUT=${KROXYLICIOUS_CHECKOUT:-${SCRIPT_DIR}/..}
+
+KAFKA_VERSION=${KAFKA_VERSION:-$(mvn -f ${KROXYLICIOUS_CHECKOUT}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=kafka.version -q -DforceStdout)}
+STRIMZI_VERSION=${STRIMZI_VERSION:-$(mvn -f ${KROXYLICIOUS_CHECKOUT}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=strimzi.version -q -DforceStdout)}
+export KAFKA_VERSION STRIMZI_VERSION
+
+runDockerCompose () {
+  docker-compose -f ${PERF_TEST}/docker-compose.yaml "${@}"
+}
+
+doCreateTopic () {
+  local TOPIC
+  ENDPOINT=$1
+  TOPIC=$2
+  docker run --rm --network perf-tests_perf_network quay.io/strimzi/kafka:${STRIMZI_VERSION}-kafka-${KAFKA_VERSION}  \
+      bin/kafka-topics.sh --create --topic ${TOPIC} --bootstrap-server ${ENDPOINT} 1>/dev/null
+}
+
+doDeleteTopic () {
+  local ENDPOINT
+  local TOPIC
+  ENDPOINT=$1
+  TOPIC=$2
+  docker run --rm --network perf-tests_perf_network quay.io/strimzi/kafka:${STRIMZI_VERSION}-kafka-${KAFKA_VERSION}  \
+      bin/kafka-topics.sh --delete --topic ${TOPIC} --bootstrap-server ${ENDPOINT}
+}
+
+
+warmUp() {
+  echo -e "${YELLOW}Running warm up${NOCOLOR}"
+  producerPerf $1 $2 1000 > /dev/null
```

Comment:
Warmp up still needs a bigger amount of messages to be sent. We used to send 3000000 records to be at least 30 secs stabilizing the system.